packages/client/components/AnalyticsPage.tsx

```diff
@@ -157,6 +163,23 @@ const AnalyticsPage = () => {
       )
     }, TIME_TO_RENDER_TREE)
   }, [isSegmentLoaded, pathname])
+
+  useEffect(() => {
+    if (!datadogEnabled) {
+      return
+    }
+
+    const {viewerId} = atmosphere
+    if (viewerId) {
+      datadogRum.setUser({
+        id: atmosphere.viewerId,
```

Comment:
Ideally, we'd pass the `email` here, too. That's data I can get back from the `AnalyticsPageQuery`, defined above. But that's requiring a bit more refactoring, so I'm going to break it into a separate PR for clarity. I think that change will also potentially fix a few other bugs that I'll describe on the next PR.